server/src/api/services/external/cml.service.ts

```diff
@@ -1,6 +1,7 @@
 import axios from 'axios';
 import { CML } from '../../constants';
-import { ServerError } from '../../errors';
+import { NotFoundError, ServerError } from '../../errors';
+import cheerio from 'cheerio';
```

Comment:
Can you move cheerio import to below axios? I try to keep all npm imports above local imports